src/Configurations/Fields/Sections/Synchronization.js

```diff
@@ -14,44 +14,55 @@ import {
   Select,
 } from '@folio/stripes-acq-components';
 
-import { TIME_UNITS } from '../../../const';
+const validateNumber = value => {
+  const integerAccept = /\d+/g;
```

Comment:
does it validate the value as intended?
![image](https://user-images.githubusercontent.com/72964370/124091455-88183180-da5e-11eb-9762-1a183e30e063.png)

Reply:
IMHO the global flag (`g`) is redundant here.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp/test#using_test_on_a_regex_with_the_global_flag

Reply:
I've mistaken on my test in console!
And the global flag is the source of my mistake:

Before I hit Enter:
![image](https://user-images.githubusercontent.com/72964370/124094322-594f8a80-da61-11eb-83b9-692917f4a51a.png)

After:
![image](https://user-images.githubusercontent.com/72964370/124094443-784e1c80-da61-11eb-9142-85de22e7c71c.png)

That's the catch of global flag: `test` on every run will process the string not from beginning, but from the point it stopped last time.

Reply:
And without global flag the regex won't work as intended here:
![image](https://user-images.githubusercontent.com/72964370/124095127-0f1ad900-da62-11eb-9268-250cac3bf3ab.png)

So I guess no changes needed.
Sorry for disturbance )